lib/mono/cli/publish.rb

```diff
@@ -161,11 +194,23 @@ def commit_changes(packages)
             ["Publish package #{only_package.next_tag}", standard_message]
           end
         run_command "git add -A"
+        rollback << [
+          "## Removing release commit",
```

Comment:
This is for unstaging the staged files?

Reply:
Yep :) There's also a `git restore :/` in a previous step that undoes the changes to the unstaged files.